cypress/README.md

```diff
@@ -0,0 +1,114 @@
+## Install
+
+You need to have the project up and running on port 4200 ⇒ https://github.com/usealto/assessment-front/blob/develop/README.md
+
+## Run Cypress
+
+If it’s the first time, you need to ask a coworker for the environnement file ⇒ `cypress.env.json` 
+
+### With details
+
+1. Start the project
+2. Open Cypress
+3. Click on a test to run it
+
+Start a local server using 
+
+```jsx
+npm run start
+```
+
+---
+
+Open Cypress Desktop app using 
+
+```jsx
+npx cypress open
+```
+
+Then you can select type testing (here we choose E2E Testing)
+
+![Capture d’écran 2023-10-09 à 16 53 52](https://github.com/usealto/assessment-front/assets/107506961/d1cef231-ad17-4406-8362-e52181dd6128)
+
+After that you must choose a browser to execute your tests (here we take Chrome)
+
+![Capture d’écran 2023-10-09 à 16 54 02](https://github.com/usealto/assessment-front/assets/107506961/4440f098-d5f8-450c-8be0-13bcad899c4b)
+
+And there you go, you can run all your spec files manually just by clicking them
+
+![Capture d’écran 2023-10-09 à 16 55 11](https://github.com/usealto/assessment-front/assets/107506961/6d3d38f0-6c12-436a-a4a8-5a0b8fc6c415)
+
+---
+
+### Global Run
+
+In case you want to run all your tests at once and get a global picture you can use
+
+```jsx
+npm run cypress:run
+```
+
+## Write Cypress Tests
+
+### With details
```

Comment:
is there another way of writing test ? if not, we could remove this subtitle i believe

Reply:
done